src/main/java/Edit_Group_Profile_Use_Case/EditGroupProfileUseCase.java

```diff
@@ -0,0 +1,22 @@
+package Edit_Group_Profile_Use_Case;
+import Entities.Group;
+
+import java.util.HashMap;
+
```

Comment:
I'm kind of confused by the purpose/role of this class in the code and where it is in the clean architecture? Since the use case is played out by the EditGroupProfileInteractor and I don't see this class being used anywhere else? Maybe I am missing something. We should discuss this detail!

Reply:
Yes, I'm not sure this is necessary since it does not seem like you are calling this anywhere outside of this class. I originally had a similar class, but turned it into the interactor which now handles all of the necessary functions when the use case is called.